src/main/kotlin/org/axonframework/intellij/ide/plugin/markers/handlers/CommonHandlerMethodLineMarkerProvider.kt

```diff
@@ -29,7 +29,7 @@ import org.axonframework.intellij.ide.plugin.util.creatorResolver
  * Provides a gutter icon on all generic handler methods
  */
 class CommonHandlerMethodLineMarkerProvider : AbstractHandlerLineMarkerProvider() {
-    private val blacklistedTypes = listOf(MessageHandlerType.DEADLINE, MessageHandlerType.COMMAND, MessageHandlerType.COMMAND_INTERCEPTOR)
+    private val blacklistedTypes = listOf(MessageHandlerType.DEADLINE)
```

Comment:
This class marks any `@MessageHander` annotated method, unless that handler type is in the list.

Before this PR, a command handler had a specific one, but with the interceptors gone it's not needed. Aditionally, in the future, the interceptors will be generic for any type of message and implemented in this provider.
The reason deadline is still here is because this is a specific one (`DeadlineHandlerMethodLineMarkerProvider`) because it uses the `DeadlineManagerReferenceResolver` instead of the `MessageHandlerResolver`

Reply:
I left it a list, because maybe we need different MarkerProviders later after all

Reply:
Thanks for the explanation; it makes sense to me!